android/engine/src/main/java/org/smartregister/fhircore/engine/ui/components/PaginatedRegisterViews.kt

```diff
@@ -178,32 +181,64 @@ fun PaginatedRegister(
   nextButtonClickListener: () -> Unit,
   modifier: Modifier = Modifier
 ) {
-  Column(modifier = modifier.fillMaxSize()) {
-    if (showResultsCount) {
-      SearchHeader(resultCount = resultCount)
-    }
-    Box(
-      contentAlignment = Alignment.TopCenter,
-      modifier = modifier.weight(1f).padding(4.dp).fillMaxSize()
-    ) {
-      if (loadState == LoadState.Loading) {
-        CircularProgressBar()
-      } else {
-        if (resultCount == 0 && showResultsCount) {
-          NoResults(modifier = modifier)
+  Column(modifier = modifier.fillMaxWidth().height(200.dp)) {
+    ConstraintLayout(modifier = Modifier.fillMaxSize()) {
+      val (_top, _body, _bottom, _b2) = createRefs()
```

Comment:
Remove underscore in destructed data class properties names unless those are the actual names. Rename **_b2** appropriately. Underscores are conventionally used in field properties to denote them as private. Sometimes they are used as placeholders for unused properties.

Reply:
updated